### ISSUE TITLE
Getting Error: Vuetify is not properly initialized on trying example notebook

### ISSUE THREAD

**Issue by tdeboer-ilmn** [voila-dashboards/voila-vuetify#43]
I tried the example notebook in my setup and am seeing the error

Error: Vuetify is not properly initialized

I am using the following versions of ipyvuetify and voila:

```
  bluebee@bbench:~/.applications$ pip3 freeze|egrep 'voila|vue'
  ipyvue==1.5.0
  ipyvuetify==1.6.2
  voila==0.2.7
  voila-vuetify==0.5.1
```

I'm uisng JupyterLab 3.0.8 and have activated the extensions as per the documentation, and this setup used to work correctly in JupyterLab < 3 so maybe something has changed in the setup?

Thanks!

![image](https://user-images.githubusercontent.com/72627522/108923064-fc445380-75ec-11eb-853a-e8e2eccbf4ba.png)

**Comment by umrashrf** [comment]
I have the same error.

```
$ cat requirements.txt
jupyterlab
voila-vuetify
bqplot
ipyvuetify
```

**Comment by mariobuikhuizen** [comment]
Thanks for reporting! Will be fixed with #44.

**Comment by mariobuikhuizen** [comment]
#44 is released in v0.5.2
